### PR TITLE
Exposing slack API for fetching user's slack avatars

### PR DESCRIPTION
**Pull request by kkateq**
So the slack API to get users list called by prefetch job on the nodeJS side and users slack data are cached to redis. Since the fetch of each slack profile is an expensive operation. Users filtered now only whether they are bots or not. We might want to filter even more by some additional criteria.

Then on the first tanto load, slack data fetched from redis along with the ono users fetch saga. 

Only slack id, email, timezone offset and avatar profile properties are used. 

Ono users and slack profiles are matched by emails. We might need to expand the logic of matching user's slack profiles to ono, because not all ono users got matched by email.

Prefetch slack profiles property job would run at the beginning of every month(customizable).

SLACK CLIENT ID AND SECRET ID, SLACK token should be added to `.env`, ask me if you need ones. 

Not all code covered with unit tests though, but I'll add down the way, did get some problems with mockery cache.

`TestAvatar`, `UserAvatar` components were removed.

### CODE REVIEW

File: src/client/components/Avatar/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+const avatarStyle = {
+  borderRadius: '20%',
+  float: 'left',
+  display: 'table-column',
+  height: '40px',
+  width: '40px',
+  color: 'rgba(157, 156, 156, 0.58)',
+}
```
Comment (henriksunity) [inline src/client/components/Avatar/index.js]: Is there a compelling reason for not placing it in a css and using a className for it instead?

**Comment by wsprent** [comment]
What purpose does the empty file at `src/server/routes/index.js` serve?

File: src/client/components/Avatar/index.js (hunk 1 above)
Comment (kkateq) [inline src/client/components/Avatar/index.js]: Regarding CSS topic, I guess lot's of you confused. So no work related CSS modularity done in the project yet, for the reason that styling and design haven't been finalized yet and maintaining css files adds on component development time. 

Loading `.css` file for the react component doesn't guarantee you that another react component accidentally will not override your styles. To get this right, we will introduce CSS plugins e.g. postCss, or will continue using inline styles.

There is no performance impact on using inline styles either. So if you sure there is no style which can override your one then move styles to css files in advance, if you are afraid of issues, leave them inline.

**Comment by kkateq** [comment]
>What purpose does the empty file at src/server/routes/index.js serve?

**Comment by kkateq** [comment]
>What purpose does the empty file at src/server/routes/index.js serve?
Though that wasn't finished. We'll probably get more routes on nodeJS side, so I wanted to move routes from `server/index.js` to this one for clarity.

**Pull request merged**